AutomatedTesting/Passes/MainPipeline.pass

```diff
@@ -205,6 +205,99 @@
                         }
                     ]
                 },
+
+                {
+                    // NOTE: HairParentPass does not write into Depth MSAA from Opaque Pass. If new passes downstream
+                    // of HairParentPass will need to use Depth MSAA, HairParentPass will need to be updated to use Depth MSAA
+                    // instead of regular Depth as DepthStencil. Specifically, HairResolvePPLL.pass and the associated 
+                    // .azsl file will need to be updated.
+                    "Name": "HairParentPass",
+                    // Note: The following two lines represent the choice of rendering pipeline for the hair.
+                    // You can either choose to use PPLL or ShortCut and accordingly change the flag 
+                    // 'm_usePPLLRenderTechnique' in the class 'HairFeatureProcessor.cpp'
+//                    "TemplateName": "HairParentPassTemplate",
```

Comment:
Should this line be removed?

Reply:
no, it's another option to be tested / used the exists and is fully functional.
Removing it will make sure that it will never get used and this is why I added the remark lines above